orga/tests/integration/components/routes/authenticated/campaign/report-test.js

```diff
@@ -12,69 +12,94 @@ module('Integration | Component | routes/authenticated/campaign/report', functio
     store = this.owner.lookup('service:store');
   });
 
-  test('it should display campaign details', async function(assert) {
+  test('it should display campaign name', async function(assert) {
     // given
     const campaign = store.createRecord('campaign', {
-      id: 1,
       name: 'campagne 1',
     });
+    this.set('campaign', campaign);
+
+    // when
+    await render(hbs`<Routes::Authenticated::Campaign::Report @campaign={{campaign}}/>`);
 
+    // then
+    assert.contains('campagne 1');
+  });
+
+  test('it should display campaign code', async function(assert) {
+    // given
+    const campaign = store.createRecord('campaign', {
+      code: '1234PixTest',
+    });
     this.set('campaign', campaign);
 
     // when
     await render(hbs`<Routes::Authenticated::Campaign::Report @campaign={{campaign}}/>`);
 
     // then
-    assert.dom('.campaign-details-header__title').hasText('campagne 1');
+    assert.contains('1234PixTest');
   });
 
-  module('Campaign details header', function() {
-    test('it should display the campaign report', async function(assert) {
+  module('When there is some results', function() {
+    test('it should display campaign participants number', async function(assert) {
       // given
-      const campaignReport = store.createRecord('campaignReport', {
-        id: 1,
+      const campaign = store.createRecord('campaign', {
         participationsCount: 10,
-        sharedParticipationsCount: 4,
       });
+
+      this.set('campaign', campaign);
+
+      // when
+      await render(hbs`<Routes::Authenticated::Campaign::Report @campaign={{campaign}}/>`);
+
+      // then
+      assert.contains('10');
+    });
+
+    test('it should display campaign shared participations number', async function(assert) {
+      // given
       const campaign = store.createRecord('campaign', {
-        id: 1,
-        name: 'campagne 1',
-        campaignReport,
-        code: '1234PixTest',
-        type: 'ASSESSMENT',
+        sharedParticipationsCount: 4,
       });
-
       this.set('campaign', campaign);
 
       // when
       await render(hbs`<Routes::Authenticated::Campaign::Report @campaign={{campaign}}/>`);
+
       // then
-      assert.dom('.campaign-details-header-report__info:nth-child(1) .campaign-details-content__text').hasText('1234PixTest');
-      assert.dom('.campaign-details-header-report__info:nth-child(2) .campaign-details-content__text').hasText('10');
-      assert.dom('.campaign-details-header-report__shared .campaign-details-content__text').hasText('4');
+      assert.contains('4');
     });
+  });
 
-    test('it should display - instead of 0 for the campaign report', async function(assert) {
+  module('When there is no results', function() {
+    test('it should display - when participationsCount is 0', async function(assert) {
```

Comment:
**suggestion (non-blocking) :** `it should display '-' when no one participated`